src/bot/conversation/timezone.py

```diff
@@ -4,24 +4,24 @@
     KeyboardButton,
     ReplyKeyboardMarkup,
     ReplyKeyboardRemove,
-    Update,
+    Update
 )
 from telegram.ext import (
     CallbackContext,
     CallbackQueryHandler,
     ContextTypes,
     ConversationHandler,
     MessageHandler,
-    filters,
+    filters
 )
 
-from constants import callback_data, states
-from constants.timezone import DEFAULT_TIMEZONE
+from bot.constants import callback_data, states
+from bot.constants.timezone import DEFAULT_TIMEZONE
+from bot.decorators.logger import async_error_logger
+from bot.get_timezone import get_timezone_from_location, get_timezone_from_text_message, set_timezone
```

Comment:
Тут тоже давай изменим импорт
```suggestion
from bot import get_timezone
```